fixtures/ios_app_with_multi_configs/App/Project.swift

```diff
@@ -0,0 +1,40 @@
+import ProjectDescription
+
+let configurations: [CustomConfiguration] = [
+    .debug(name: "Debug", xcconfig: "../ConfigurationFiles/Debug.xcconfig"),
+    .release(name: "Beta", xcconfig: "../ConfigurationFiles/Beta.xcconfig"),
+    .release(name: "Release", xcconfig: "../ConfigurationFiles/Release.xcconfig"),
+]
+
+let settings = Settings(base: [
+    "PROJECT_BASE": "PROJECT_BASE",
+], configurations: configurations)
+
+let betaScheme = Scheme(name: "App-Beta",
+                         shared: true,
+                         buildAction: BuildAction(targets: ["App"]),
+                         runAction: RunAction(configurationName: "Beta", executable: "App"))
+
+let project = Project(name: "MainApp",
```

Comment:
During my testing, I added some configuration overrides for `Target` too. Which all works, might be a good idea to add them to this fixture to avoid regressions on that.

```swift
let project = Project(name: "MainApp",
                      settings: settings,
                      targets: [
                          Target(name: "App",
                                 platform: .iOS,
                                 product: .app,
                                 bundleId: "io.tuist.App",
                                 infoPlist: "Support/App-Info.plist",
                                 sources: "Sources/**",
                                 dependencies: [
                                    .project(target: "Framework1", path: "../Framework1"),
                                    .project(target: "Framework2", path: "../Framework2"),
                                ],
                                 settings: Settings(base: [
                                    "TARGET_BASE": "TARGET_BASE",
                                ], configurations: [
                                    .debug(name: "Debug", xcconfig: "../ConfigurationFiles/Target.Debug.xcconfig"),
                                    .release(name: "Release", xcconfig: "../ConfigurationFiles/Target.Release.xcconfig"),
                                ])
                          ),
                          Target(name: "AppTests",
                                 platform: .iOS,
                                 product: .unitTests,
                                 bundleId: "io.tuist.AppTests",
                                 infoPlist: "Support/AppTests-Info.plist",
                                 sources: "Tests/**",
                                 dependencies: [
                                     .target(name: "App"),
                          ]),
                      ], schemes: [betaScheme])
```

<img width="475" alt="image" src="https://user-images.githubusercontent.com/1382565/62042965-383bea80-b1f7-11e9-9e2f-ea96a4c2b53b.png">